csp/contrib/rate_limiting.py

```diff
@@ -28,9 +28,14 @@ def build_policy(self, request: HttpRequest, response: HttpResponseBase) -> str:
             return ""
 
         report_percentage = policy.get("REPORT_PERCENTAGE", 100)
-        include_report_uri = random.randint(0, 100) < report_percentage
-        if not include_report_uri:
-            replace["report-uri"] = None
+        remove_report = random.randint(0, 99) >= report_percentage
```

Comment:
Maybe it's my brain but I find this way around harder to track in my head. Maybe worth a comment to clarify?

Reply:
Ha, that's interesting. I was trying to remove the `if not include` and simplify to `if remove` since the code block inside the conditional removes the reporting. But perhaps I'm overlooking something.